content/project/opees/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Opees"
+date: 2020-03-12T09:22:18-04:00
+logo: "/images/research/opees.jpeg"
+tags: ["Modeling"]
+homepage: ""
+facebook: ""
+linkedin: ""
+twitter: ""
+youtube: ""
+funding_bodies: ["itea3"]
+eclipse_projects: ["modeling.mdt.papyrus","polarsys.capella"]
+project_type: success
+summary: "Open Platform for the Engineering of Embedded Systems"
+---
+Source: <https://itea3.org/magazine/34/november-2019/itea-success-story-opees.html>
+
+**The ITEA 2 project OPEES stood at the inception of two important trends: open collaboration with open source in industry and open source tools for model-based systems engineering (MBSE). Neither of these trends were well developed in 2009, but almost 10 years later, and with acceleration through the OPEES project, we benefit from both good open source MBSE tools and many open collaboration initiatives in industry. OPEES was both a pioneer and a catalyst in this evolution.**
+
+The ITEA 2 OPEES project was created to develop an open source platform for software tools to support engineering technologies for embedded systems and to secure the competitiveness and development of the European software industry. One key requirement, brought by Airbus, was to be able to use tools for more than 50 years, during the complete lifetime and duration of support of an aircraft programme. During the project that ended in 2012, the 28 partners not only developed and significantly improved existing open source projects such as Frama-C, Eclipse Papyrus and others, but also defined the governance and the structure for a sustainable organisation to gather an ecosystem of both developers and users.
+<center><img src="opees-story.png" width="400"></center>
```

Comment:
You should use the [figure shortcode](https://gohugo.io/content-management/shortcodes/#figure) to display images here.

e.g.
```{{< figure src="/images/donate.jpg" alt="Donate now!" class="vertical-align" width="400">}}```